Adding acceptance tests instructions to the acceptance-tests doc

computate opened the pull request:
- For the items that are OpenShift related, there are "Acceptance tests:" instructions on what to do following each of the steps to review.
- For the items that may not be OpenShift related, I marked them with "Acceptance tests: requires investigation".

joachimweyl commented on acceptance-tests.md:
```diff
@@ -148,12 +175,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
         3. [Click here to view the memory usage of projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22namespace:container_memory_usage_bytes:sum%22%7D%5D).
         4. [Click here to view the cpu usage of the projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum%22%7D%5D).
-        5. [Click here to show the projects using the top 5 cpu usage at each point in time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22topk(5,%20(1%20-%20avg(rate(node_namespace_pod_container:container_cpu_usage_seconds_total:sum%7B%7D%5B$__rate_interval%5D))%20b ...
+        5. [Click here to show the projects using the top 5 CPU usage at each point in time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22topk%285,%20%281%20-%20avg%28rate%28node_namespace_pod_container:container_cpu_usage_seconds_total:sum%7B%7D%5B$__rate_interval% ...
```
something about the new section broke the numbering please double-check that you have the expected carriage returns etc.

computate replied:
Thanks for the great feedback here Kim, I have resolved the numbering issues.

joachimweyl commented on acceptance-tests.md:
```diff
@@ -148,12 +175,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
     * <span style="text-decoration:underline;">Steps</span>
         1. Administrator logs into the associated XDMoD instance and views reports.
+    * <span style="text-decoration:underline;">Acceptance tests:</span>
         2. [Click here to view the ACM Observability Grafana dashboards](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/). These dashboards provide insights into Control Plane Health, Optimization, Capacity, Utilization and more. You can change the timespan in the top right to show results in terms of minutes, hours, days, months or years.
```
did you want this to be 2. or 1. again? I also don't think that HTML styles come through for Markdown.

computate replied:
Thanks for the great feedback here Kim, I have resolved the numbering issues.

joachimweyl commented:
The formatting and wording look good, I can't speak to the links as I don't have access.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Steps
         1. A prospective user follows the steps documented in [https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/](https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/) to create an account on the NERC.
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate). Note the identities for the given user, specifically one that starts with `github:`, like `github:15114160`.
```
before step 1, you have to check the MSS Account portal to see that the MGHPCC-SS Account has been created, and view the assigned username.  There can be a delay of days between when the account is submitted and when it is created, which varies depending on the type of account approval required by the NERC.  Suggest a particular test user accountname (requires functional email) here for clarity.

Under what conditions are there multiple identities expected for a given user.  The example should include the case of a user with a harvard or BU email, because that will be the usual case, not the github case.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+        2. For a given users identity, `github:15114160` for example, [Click here to check that the given identity is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~Identity&name=github%3A15114160).
 2. **Remove a user**
     * _As an administrator, I should be able to remove a user, labels association with the user, and user identity from the cluster._
        * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
```
Unclear which step you are saying is unnecessary.  The use case is to remove a user, and the user probably does have projects and allocations associated with their account.  All user information should be removed for the acceptance test to pass.

computate replied:
@knikolla I see that you added these comments about this step being unnecessary in this "NERC/OpenShift Acceptance Tests" document: https://docs.google.com/document/d/18AUzxuFWKeNOO6UFgze6Tm1mF1Gz-UIMj8Wqu6lyxcc/edit
Would you be able to help us clarify this use case, "Remove a user"?

hpdempsey replied:
Please revise the parenthetical text in this section and update the use case words quoted here in italics  to match the change in your other approved PR revising the use case for removing a user

hpdempsey replied:
@computate ^

hpdempsey commented on acceptance-tests.md:
```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
        * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A NERC admin uses a private runbook to disable/ deactivate a user from KeyCloak.
```
Provide a pointer to directions and/or an example.  Note this step needs pass/fail too, if it hasn't already been done.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Request a new account**
     * _As a new user, I should be able to create an account for myself in the NERC._
```
The acceptable use notice acceptance tests comes before this one.  The current expectation is that the users see an acceptable use notice when they sign up for the NERC account through the account portal.  @knikolla shoudl be able to specify on what screen this is suppposed to show up, so it can be verified with a manual web browser check.  This should work regardless of whether someone is requesting an account via their university, Git or other affilliation.

hpdempsey replied:
@knikolla do you know where the AU notice will be displayed now?  It is important to document this  the acceptance tests.

hpdempsey replied:
We do not need to document location on the screen, but acceptance test needs to verify that it is visible.  Verify this when we walk through acceptance tests (or note that it is not possible to find).

hpdempsey commented on acceptance-tests.md:
```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
         3. To approve the request, NERC admin assigns the user to the PI role on KeyCloak’s user management.
         4. NERC admin approves the request and responds with a ticket reply.
         5. To remove the PI role, just reverse the step iii. And send out an email to the user informing about it.
+    * Acceptance tests: requires investigation
```
What investigation is happening?

hpdempsey replied:
add/remove admin priv. is missing as an acceptance test

hpdempsey replied:
Text has been revised and now includes appropriate steps.    Acceptance test step "e"  still needs edits, as it is the acceptance test for deleting PI role from a user, but the check doesn't show that.  Reference to step "iii" in earlier text should be step "c"

hpdempsey replied:
@computate ^

hpdempsey commented on acceptance-tests.md:
```diff
@@ -36,7 +43,7 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 1. **Add a new project**
     * _As a user who is a PI, I should be able to create a project by._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
```
Once step 4 is completed, where do you look in coldfront to see what the project name is supposed to be?  What do you check in ACM to verify that the project has been created and is associated with the PI who created it?  Is there any delay expected between steps 4 and steps 5 completing?

hpdempsey replied:
addressed with revisions

hpdempsey commented on acceptance-tests.md:
```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
         1. A Coldfront admin can navigate to the project
             1. They can archive a project and expire all associated allocations by clicking ‘archive project’ by navigating to the project and clicking ‘archive project’.
             2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
         2. Disabling an allocation will delete the associated OpenShift namespace, which differs from OpenStack behavior which simply disables the project.
```
What is the delay (if any) between Coldfront delete and OpenShift delete?  How do you show that the Coldfront delete action was taken.  (OpenShift delete check is already shown, but if there's a possible delay, that needs to be noted).  Do you need to release the resources before deleting the project in OpenShift, or does it all happen automatically (storage, for example).  How does one verify that the resourcees were actually released?

hpdempsey replied:
No delay according to Kristi.  Coldfront display of user's projects updates when change is made withing Coldfront, and steps required to check this on OpenShift are covered in acceptance tests.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Acceptance tests:
+        1. For a given project named `myproject` for example, [Click here to check that the given project is no longer listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=project.openshift.io%7Ev1%7EProject&name=myproject).
 3. **Manage a project as a PI.**
     * _As a PI, I should be able to manage and share my project with others on the team, but no one except the PI and the administrator should be able to remove the project._
```
Is it really true the admin can delete the project?  That is not in the use case.

hpdempsey replied:
Description is correct.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A PI can add keycloak users to a ColdFront project under the ‘users’ section in the given project ([https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project))
         2. From here, a PI can set the user to a particular role. It seems the ‘manager’ role is the one that lets users create and remove allocations by delegating PI role/responsibilities. Some other roles should be set for other general users who are only going to use the allocated NERC resources.
```
Please be more specific than "some other roles"  I think there re only PI, manager and user  roles in ColdFront.  You explain how to check roles from ColdFront are present in keycloak.  Is there any check that needs to be done as an acceptance test to verify that the "manager" role can add and delete users, for example?

hpdempsey replied:
Acceptance test OK as written.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Acceptance tests:
+        1. For a given project named `myproject`, and a given user named `myuser`, and a given role named `manager` for example, [Click here to check that the given project contains a RoleBinding with a "Role ref" of `manager`, a "Subject kind" of `User`, and a "Subject name" of `myuser` in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/search/ns/mypro ...
 
 
```
Use case 2.1 about a user being able to add new namespaces is not covered here.  Is it covered in the OpenShift basic tests?

hpdempsey replied:
A non-admin user cannot request a new namespace.  The steps for a PI to request the namespace are documented now, so this is resolved for the acceptance tests.  The non-admin namespace request function needs to be removed from the use case document.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -66,10 +79,12 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 1. **Set and modify quotas for projects**
     * _As an administrator of the cluster, I should be able to set and modify compute, storage and object counts quotas for any project._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
```
This is not clear about how changes to allocations are made.  They are made in ColdFront and have to be passed to OpenShift.  I don't believe they will be made in OpenShift directly, because the two would get out of sync.

hpdempsey replied:
Description was revised to make this clear.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -66,10 +79,12 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
             1. For setting attributes, adding a new allocation attribute triggers a call to the acct-mgt service endpoint “/projects/{project_id}/quota".
+    * Acceptance tests:
+        1. You can explore quotas from within the Observability dashboard. For a given project named `myproject`, and resource `limits.cpu` for example, [Click here to check the "value" for the type=hard (max limit) and type=used (current value)](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemp ...
 
```
The acceptance test should walk through the change on the ColdFront side, verify that, and then the instructions about how to see the appropriate quota has changed in Obeservatorium are the OpenShift side.   Same approach as the previous use cases that are spread across two tools.

hpdempsey replied:
revisions made

hpdempsey commented on acceptance-tests.md:
```diff
@@ -78,9 +93,11 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Steps
         1. After a user is added, an admin can go to the user ‘actions’ tab and set their role to ‘manager’ or ‘user’.
             1. [https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project)
+    * Acceptance tests:
```
Same question about delays for this change to get from ColdFront to OpenShift and how to test the use case.

hpdempsey replied:
@knikolla says there will be no delay for this update to propogate to OpenShift.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -89,9 +106,10 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Steps
         1. Administrator/user accesses documentation at:
             1. [https://nerc-project.github.io/nerc-docs/](https://nerc-project.github.io/nerc-docs/)
+    * Acceptance tests: requires investigation
```
Just requires listing the locations of all ops docs.  Besides NERC repo you have listed, there is Red Hat support documentation for what we installed (ACM, OpenShift etc), and the documentation and code for what was done with ColdFront and XDMOD.  You will have to ask about those two at the NERC meeting.  So acceptance test is just verifying that each part of the NERC builld is covered by the documentation links in this section and the links are live.  You may have to say something about credentials too, since Red Hat stuff sometimes requires a license.

joachimweyl replied:
[ColdFront Upstream](https://github.com/ubccr/coldfront/blob/master/README.md)
[What is NERC's ColdFront](https://github.com/nerc-project/nerc-docs/blob/main/docs/get-started/get-an-allocation.md)

[XDMoD Upstream](https://github.com/ubccr/xdmod/blob/xdmod10.5/README.md)
[MOC XDMoD OpenStack Readme](https://github.com/CCI-MOC/xdmod-cntr/blob/main/README.md)
[NERC XDMoD OpenShift Readme](https://github.com/OCP-on-NERC/xdmod-openshift-scripts/blob/main/README.md)
[NERC XDMoD OpenShift Integration](https://github.com/OCP-on-NERC/docs/blob/main/xdmod_integration.md)

hpdempsey replied:
@computate @joachimweyl The link currently referenced here does not include pointers to all the docs in Kim's list.  @joachimweyl are there plans to add these docs to that link?  Or do we need to include pointers to them in the acceptance tests?

joachimweyl replied:
@heidi I will work with @jtriley to ensure this list gets into their runbook.
Hello @jtriley the list we talked about in the meeting today is [2 comments up](https://github.com/OCP-on-NERC/docs/pull/37#discussion_r1024416780). Please let me know how I can help ensure they get where they need to be. If the runbook is in a location I can update I would be happy to create a PR to update it.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -100,12 +118,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Netbox
+    * Acceptance tests: requires investigation
```
This needs to be assigned to Justin.  We have no knowledge of it at Red Hat.

hpdempsey replied:
@jtriley  Do you have a pointer to the Netbox information for checking on this?

hpdempsey replied:
Documentation updated.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -100,12 +118,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
     * _As an administrator of the cluster, I should be able to view and track the list of faulty nodes that need to be replaced._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Nagios or refer to notes in Netbox
```
Is there anything in ACM that allows you to track a list/re[rt of "faulty" nodes?  If so, that is relevant here.  Otherwise, this is another item for Justin.  At the very least, you want to make sure that we explain how to keep the OpenShift understanding of what working and infra servers are available in agreement with what hardware is actually available, and how to check on this.

hpdempsey replied:
@computate what was the answer on whether this function will be provided in ACM?

hpdempsey commented on acceptance-tests.md:
```diff
@@ -114,8 +134,10 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Upgrade documentation available in the documentation repository
+    * Acceptance tests:
```
Are there any steps to upgrading that are unique to NERC and not documented in the general Red Hat product upgrade docs.  (There are, and that's what should be documented here, in addition to the standard Red Hat upgrade docs links.)

hpdempsey replied:
Consensus was that standard upgrade docs will suffice.  Chris added a link to ACM functions that are useful for checking versions and upgrades.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -124,20 +146,25 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Acceptance tests:
+        1. [Click here to check that alerts are sent to `alerts-nerc-ocp` channel in Slack](https://app.slack.com/client/T14CVG7L1/C044ZCTUWCV/thread/C027TDE52TZ-1666641224.695039).
 2. **Logging**
     * _As an administrator of the cluster, I should be able to track all the events in the cluster using the logging system in OpenShift._
```
Log archiving and rollover is something that often makes acceptance tests fail. after systems have been running for a while  Please document how to check on log storage space consumed vs. available using appropriate dashboards or commands.

hpdempsey replied:
@computate This documentation still needs to be added.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -124,20 +146,25 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Acceptance tests:
+        1. [Click here to visit the cpu usage logs for grafana](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22grafana%5C%22%7D%22%7D%5D).
+        2. [Click here to visit the cpu usage logs for loki](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22openshift-operators-redhat%5C%22%7D%22%7D%5D).
 
```
Good start on the  "other" logs list.  I think there are still more needed--Vault and what else?   Make sure we have a complete list of logs that are needed for various types of ops debugging and where/how to look for them.

hpdempsey replied:
@computate This info still needs to be added.

hpdempsey replied:
Acceptance test document updated.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -146,14 +173,16 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 1. **Track/report usage of the cluster**
     * _As an administrator of the cluster, I should be able to view daily, weekly and monthly reports of the cluster infrastructure utilization._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
```
Do we need to check that XDMOD and ACM usage totals match, and if so how do we do that?  Some of this will need to be assigned to Rob.

hpdempsey replied:
@computate I think this is an engineering task for you to do once XDMod info is available, but it does not have to gate acceptance tests.

hpdempsey commented on acceptance-tests.md:
```diff
@@ -124,20 +146,25 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+        2. [Click here to visit the cpu usage logs for loki](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22openshift-operators-redhat%5C%22%7D%22%7D%5D).
 
 
 ## Reporting
```
Data Management section follows Reporting, but is missing here.  This is basically how you check that archiving is done and that data that is supposed to be available is still available for the required amount of time (60 days or 90 days)

hpdempsey replied:
Data Management use cases still need to be covered.

hpdempsey replied:
Data Management added to acceptance tests.

hpdempsey reviewed (commented):
Please use the numbering scheme from the use cases for the acceptance testss.
Please add text for pass/fail criteria for each use case.  (Suggested screen capture examples to Chris offline).  For example, the "Add a user" use case is successful if the username specified by the user who requested their account is displayed in screen XXX of ACM as shown in this screen capture (link to filename).
Please add info at the beginning specifying the accounts/access needed to see all the results mentioned for acceptance tests in the use cases.

computate commented:
@hpdempsey I have now followed the number scheme, added better pass/fail criteria, and added screenshots. I wasn't able to add info at the beginning specifying the accounts/access needed, because they will not be able to provide a test account.

larsks commented on acceptance-tests.md:
```diff
@@ -2,32 +2,62 @@ These acceptance tests are based on the [NERC Operational Use Cases](https://doc
+        1. Check that an OpenShift User exists, with access to the project allocations:
+            1. Create a new user account following the documentation.
+            2. For a given username of `nerc-test-account` for example, check that the given username is listed in the `oc` CLI:
+                1. `oc get User/nerc-test-account`<br/> ![NERC test account](img/acceptance-tests/managing-users-1.1.2-user-exists.png)
```
I would like to see terminal examples included as text, rather than as images. This often makes them more legible, and it certainly makes them more accessible e.g. to someone using an assistive device like a screen-reader. It's also helpful if someone wants to copy/paste some portion of the example.

larsks commented on acceptance-tests.md:
```diff
@@ -36,26 +66,56 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+        2. Log into ColdFront as the user and request a project.
+        3. As the user, request an OpenShift resource allocation in ColdFront.
+        4. Log into ColdFront as an admin and approve the request.
+        5. Validate the project was create with the requested quota:
```
s/was create/was created/

larsks commented on acceptance-tests.md:
```diff
@@ -36,26 +66,56 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+        4. Log into ColdFront as an admin and approve the request.
+        5. Validate the project was create with the requested quota:
+            1. For a given project named `012345myproject` for example, check that the given project is listed in the `oc` CLI:
+                1. `oc get Project/012345myproject`<br/> ![Get project](img/acceptance-tests/get-project.png)
```
You don't need a `<br/>` tag here. You can just write:

```
1. `oc get Project/012345myproject`

    ![Get project](img/acceptance-tests/get-project.png)
```

But see the earlier comment about replacing these terminal examples with actual text. I would probably format these like this to show both the command and the resulting output:

```
1.
    ```
    $ oc get Project/012345myproject
    NAME             DISPLAY NAME   STATUS
    012345myproject                 Active
    ```
```

Which would render like this:


1.
    ```
    $ oc get Project/012345myproject
    NAME             DISPLAY NAME   STATUS
    012345myproject                 Active
    ```

larsks commented on acceptance-tests.md:
```diff
@@ -36,26 +66,56 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
-            1. They can archive a project and expire all associated allocations by clicking ‘archive project’ by navigating to the project and clicking ‘archive project’.
-            2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
+    * Steps
+        1. A ColdFront admin can navigate to the project. [See the ColdFront documentation here](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/).
```
Should these steps be written more actively? The heading hear is "Steps", but this isn't really a list of steps; in it's current form it's a list of criteria.

computate committed:
Adding acceptance tests instructions to the acceptance-tests doc

- Adding screenshots and acceptance tests for most of the cases.
- Listing the reequired access to applications for acceptance testers.
- There are followup PRs for certain acceptance tests where the words
were discussed and changed.

computate committed:
Adding the missing Data Management use cases and tests

hpdempsey reviewed (approved):
All my comments have been addressed.  In some cases NERC deferred requirements, so acceptance tests will be completed for those requirements.

larsks reviewed (approved):
Thanks for making those changes!
